src/plugins/condition/StyleRuleManager.js

```diff
@@ -120,6 +128,7 @@ export default class StyleRuleManager extends EventEmitter {
                 //Only resubscribe if the conditionSet has changed.
                 if (isNewConditionSet) {
                     this.subscribeToConditionSet();
+                    this.addBoundsAndEditSubscriptions(styleConfiguration);
```

Comment:
ensure we only add subscriptions when we have a new condition set. this goes for bounds and edit subscriptions too